test/build_branch_test.py

```diff
@@ -1,15 +1,12 @@
 #!/usr/bin/env python3
 
 import json
-import os
 from test_base import TestBase
 from pathlib import Path
 
-graph_file_path = Path(os.path.dirname(__file__)) / "data" / "build_v24_test.graphml"
-
 base = TestBase()
 base.start_server()
-print(base.warcli(f"network start {graph_file_path}"))
+base.start_network(file=Path(__file__).parent / "data" / "build_v24_test.graphml")
```

Comment:
I was hoping we could remove import pathlib and this could just be `base.start_network("build_v24_test")`, let test_base handle the rest

Reply:
we can